recipes/py-pandoc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "py-pandoc" %}
+{% set version = "2.5" %}
+{% set pypi_build = ".0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

Comment:
Ack. I should have asked this in the other one maybe. Why is conda not getting the full version string here?

Reply:
@beckermr I guess you are right. And it's better to have a full version. I ve fixed. Please PR.